docs/troubleshooting-guide/running.md

```diff
@@ -1,16 +1,16 @@
 ---
-title: Running Metabase
+title: Troubleshooting memory and JVM issues
 ---
 
-# Running Metabase
+# Troubleshooting memory and JVM issues
```

Comment:
Not sure if this is the proposed title is best, but it's for sure not a doc on running metabase